### DISCUSSION TITLE
PC replacement

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1377] [General]
Hello

Just replaced my old PC with a new one
Downloaded and installed MTB 24.1
Now, how do I copy or transfer the database from the old to the new one
Thank you
Jacky

**Comment by wolfgang-ch** [comment]
> Now, how do I copy or transfer the database from the old to the new one

Copy the whole content of the data directory https://mytourbook.sourceforge.io/mytourbook/index.php/download-install/data-directory from the old to the new PC

**Comment by gringojack** [comment]
I put aside the Mytourbook folder of the  New PC  and replaced it by the Mytourbook folder of the old PC. I got a message saying that it is not good and advisable to do so because some version was 55 instead of 54!

**Reply by wolfgang-ch** [reply]
Such a message is displayed when MT is started with a newer version of the tour database.

I recommend the following procedure on the new PC

* Uninstall MT and delete the data directory
* Install the newest version of MT on your new PC
* Run MT and close MT
* Replace the whole data directory 
* Restart MT, then it sould ask to update the tour db from 54 to 55, this is OK

**Reply by gringojack** [reply]
Thank you much Wolfgang. I downloaded 24.7 version MTB and Eclipse Adoptium Termurin.
So far it appears to run perfectly well.
